src/classes/SObjectFieldWrapper.cls

```diff
@@ -0,0 +1,29 @@
+public class SObjectFieldWrapper {
+    private final List<SObjectField> fields;
+
+    public SObjectFieldWrapper(SObjectField field) {
+        this(new List<SObjectField>{ field });
+    }
+
+    public SObjectFieldWrapper(List<SObjectField> fields) {
+        this.fields = fields;
+    }
+
+    public String Value {
```

Comment:
Yeah, makes sense!  In some instances where collections have the potential to be mutable, I've learned from Bruce to make use of properties to ensure that any call to the object is using only the most up-to-date values, but in this case there's no way for the SObjectFields to be changed after initialization.  I think a method makes more sense.